core/domain/draft_upgrade_services.py

```diff
@@ -242,6 +242,23 @@ def _convert_html_in_draft_change_list(
                 })
         return draft_change_list
 
+    @classmethod
+    def _convert_states_v52_dict_to_v53_dict(
```

Comment:
Hey @vojtechjelinek, I'm not completely sure about the changes that we have to do in `draft_upgrade_services.py` file and when should we do it?
Thanks

Reply:
Here we should invalidate all drafts that contain invalid links.

Reply:
Done